plugin/trino-delta-lake/pom.xml

```diff
@@ -261,6 +267,12 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-filesystem-azure</artifactId>
+            <scope>runtime</scope>
```

Comment:
Could it be just test scoped? Same for the okhttp dependency above?

Reply:
it is needed by tests, but also needed transitively.
we need to use runtime scope in such case, which sadly disables dep check